Buildings/Applications/package.mo

```diff
@@ -6,12 +6,8 @@ annotation (
 preferredView="info",
 Documentation(info="<html>
 <p>
-This package contains packages for different application domains.
-Currently, it contains
-<a href=\"modelica://Buildings.Applications.DataCenters\">
-Buildings.Applications.DataCenters</a>
-with models for data centers.
-It is intented that other application domains will be added in the future.
+This package contains subpackages for different application domains, such as
+data centers and district heating and cooling systems.
 </p>
```

Comment:
We may not need to change the info of this package.

Reply:
Addressed